ansible/roles/test/files/ptftests/py3/advanced-reboot.py

```diff
@@ -1453,6 +1471,25 @@ def runTest(self):
             traceback_msg = traceback.format_exc()
             self.fails['dut'].add(traceback_msg)
         finally:
+            # Restore cEOS LACP timer multiplier changes
+            if self.test_params['neighbor_type'] == "eos":
+                for neigh in self.ssh_targets:
+                    self.neigh_handle = HostDevice.getHostDeviceInstance(
+                                            self.test_params['neighbor_type'], neigh, None, self.test_params)
+                    self.neigh_handle.connect()
+
+                    raw_json = self.neigh_handle.do_cmd("show lacp interface | json")
+                    neigh_int_json = json.loads(raw_json[raw_json.find("{"):raw_json.rfind("}")+1])
+
+                    self.neigh_handle.do_cmd("config")
+                    for lag in neigh_int_json["portChannels"]:
+                        for neigh_int in neigh_int_json["portChannels"][lag]['interfaces']:
+                            self.neigh_handle.do_cmd(f"interface {neigh_int}")
+                            # Restore lacp timer multiplier to default (3)
+                            self.neigh_handle.do_cmd("lacp timer multiplier 3")
```

Comment:
Duplicate Logic for Setting and Restoring: the block that sets and resets the LACP multiplier is duplicated.
Suggest to extract the logic into a helper method.